ios/NewExpensifyShare/ShareViewController.swift

```diff
@@ -0,0 +1,260 @@
+//
+//  ShareViewController.swift
+//  RNShareMenu
+
+import Foundation
+import os
+import MobileCoreServices
+import UIKit
+import Social
+import RNShareMenu
+
+
+
+class ShareViewController: UIViewController {
```

Comment:
Just linking back to the relevant comment on this, but it's not a blocker: https://github.com/infinitered/ExpensifyApp/pull/65#discussion_r1385742814